app/players/elves/druid.py

```diff
@@ -0,0 +1,21 @@
+from app.players.elves.elf import Elf
+
+
+class Druid(Elf):
+    def __init__(
+            self,
+            nickname: str,
+            musical_instrument: str,
+            favourite_spell: str
+    ) -> None:
```

Comment:
2

Reply:
@SerhiiMar is right
try to use only one codestyle in ur code for consistency